backend/app/views/spree/admin/orders/_adjustments.html.erb

```diff
@@ -3,8 +3,8 @@
     <legend><%= title %></legend>
     <table>
       <thead>
-        <th><%= Spree.t('name')%></th>
-        <th><%= Spree.t('amount')%></th>
+        <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
```

Comment:
`:label` is already translated to "Description" though from #743.  If we want to maintain the same naming scheme from having the column as `name` we'd either need some kind of `:label_name` thing in the dictionary or to redo the other PR or change the appearance of this page